pages/Configuring/Master-Layout.md

```diff
@@ -8,37 +8,41 @@ their size.
 
 ![master1](https://user-images.githubusercontent.com/43317083/179357849-321f042c-f536-44b3-9e6f-371df5321836.gif)
 
-You can, however, resize the master window.
+However, you can resize the master window.
 
 ![master2](https://user-images.githubusercontent.com/43317083/179357863-928b0b5a-ff10-4edc-aa76-3ff88c59c980.gif)
 
 # Config
 
-*category name `master`*
+_category name `master`_
 
 | name | description | type | default |
 |---|---|---|---|---|
-| special_scale_factor | (0.0 - 1.0) the scale of the special workspace windows | float | 0.8 |
-| new_is_master | whether a newly open window should replace the master or join the slaves. | bool | false |
-| new_on_top | whether a newly open window should be on the top of the stack | bool | false |
+| pseudotile | enable pseudotiling. Pseudotiled windows retain their floating size when tiled. | bool | false |
```

Comment:
This is completely wrong. You pasted here the dwindle stuff.

Reply:
@0x4nt retard.